src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,578 @@
+    <item fieldId="3" name="MaxFPS" type="int16u"/>
+    <item fieldId="4" name="MaxHDRFPS" type="int16u"/>
+  </struct>
+
+  <struct name="VideoStreamStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <item fieldId="0" name="VideoStreamID" type="int16u"/>
+    <item fieldId="1" name="StreamType" type="StreamTypeEnum" min="0x00" max="0x03"/>
+    <item fieldId="2" name="VideoCodec" type="VideoCodecEnum" min="0x00" max="0x03"/>
+    <item fieldId="3" name="MinFrameRate" type="int16u"/>
+    <item fieldId="4" name="MaxFrameRate" type="int16u"/>
+    <item fieldId="5" name="MinResolution" type="VideoResolutionStruct"/>
+    <item fieldId="6" name="MaxResolution" type="VideoResolutionStruct"/>
+    <item fieldId="7" name="MinBitRate" type="int16u"/>
+    <item fieldId="8" name="MaxBitRate" type="int16u"/>
+    <item fieldId="9" name="MinFragmentLen" type="int16u"/>
+    <item fieldId="10" name="MaxFragmentLen" type="int16u"/>
+    <item fieldId="11" name="ReferenceCount" type="int8u"/>
+  </struct>
+
+  <struct name="ViewportStruct" apiMaturity="provisional">
+    <cluster code="0x0551"/>
+    <cluster code="0x0552"/>
+    <item fieldId="0" name="X1" type="int16u" default="0"/>
+    <item fieldId="1" name="Y1" type="int16u" default="0"/>
+    <item fieldId="2" name="X2" type="int16u" default="0"/>
+    <item fieldId="3" name="Y2" type="int16u" default="0"/>
+  </struct>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Camera AV Stream Management</name>
+    <code>0x0551</code>
+    <define>CAMERA_AV_STREAM_MANAGEMENT_CLUSTER</define>
+    <description/>
+    <client init="false" tick="false">true</client>
+    <features>
+      <feature bit="0" code="PRIV" name="Privacy" summary="Privacy supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="1" code="SCTL" name="SensorControl" summary="Sensor control supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="2" code="ADO" name="Audio" summary="Audio Stream supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="3" code="ICTL" name="ImageControl" summary="Image control supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="4" code="VDO" name="Video" summary="Video Stream supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="5" code="SNP" name="Snapshot" summary="Snapshot stream supported">
+        <optionalConform/>
+      </feature>
+      <feature bit="6" code="STOR" name="LocalStorage" summary="Local Storage available">
+        <optionalConform/>
+      </feature>
+    </features>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="MAX_CONCURRENT_VIDEO_ENCODERS" type="int8u" reportable="true" optional="true">
+      <description>MaxConcurrentVideoEncoders</description>
+      <access op="read" privilege="administer"/>
```

Comment:
Some spec questions here about whether these should be A or M, but the XML matches the spec, ok.

Reply:
Updated the Access column for all fields in the spec and incorporated in the xml.